content/scientist/Mileav Maric.md

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "name",
+  "name": "Mileva Maric",
+  "linktitle": "Mileva Maric",
+  "last": "Maric",
+  "institution_of_phd": "University of Zurich",
+  "field_of_phd": "Theoretical Physics",
+  "year_of_phd": "N/A",
+  "researchAreas": ["theoretical physics", "mathematics"],
+  "relevantCourses": ["Differential and integral calculus", "descriptive and projective geometry", "mechanics", "theoretical and applied physics", "experimental physics", "astronomy"],
+  "relevant_concepts": ["Einstein's Work"],
```

Comment:
Since there is no discussion here of her work, I would say we should remove this entry for now